template/project.yml

```diff
@@ -58,6 +58,11 @@ target-hooks:
     dir: ${docroot}/profiles/contrib/lightning
     command: npm run install-libraries
 
+# Configuration management options, such as core CM or features.
+cm:
+  features:
+    bundle: blted8
```

Comment:
Looks like this is now required, is that correct? What will this do to existing projects that don't have this defined? 

Maybe we should just use ${project.machine_name} rather than creating a new variable. Is there value in being able to customize it?